<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialNumTestDlg</class>
 <widget class="QDialog" name="SerialNumTestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>产品类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Z-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SI-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BUSBAR</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>产品条码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="operatorLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>操作员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="barCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>工装条码</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="clearRadioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>条码清零</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>自动加一</string>
   </property>
  </widget>
  <widget class="QComboBox" name="purposeComboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>中试</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>量产</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NPI</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>中试/量产</string>
   </property>
  </widget>
  <widget class="QComboBox" name="batchComboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>通用</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老化前</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老化后</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>维修</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PQC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FQC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OQA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>批次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电脑号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>测试日期</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
